dags/atd_signal_comms.py

```diff
@@ -61,6 +61,17 @@
     },
 }
 
+@task(
+    task_id="get_start_date",
+)
+def get_start_date(**context):
```

Comment:
Lots of development happening in parallel so I wanted to share that I started a [time helpers file](https://github.com/cityofaustin/atd-airflow/blob/4ca029cf9702600f51cf48c9457473233a742479/dags/utils/time.py) in another open PR. This is slightly different though and is making me think that I should have used `now()` there too.

Reply:
ah, thanks for pointing this out. i'm going to think about this some more since i'm annoyed the knack one i made is not usable in other domains.